charts/backend-service/templates/_envs.tpl

```diff
@@ -0,0 +1,2 @@
+{{- define "deployment.envs" }}
```

Comment:
Might be useful to include a comment here with some sample envs

Reply:
Even better in the readme?